File: JAVA/level1/문자열_내림차순으로_배치하기.java
import java.util.*;

class Solution {
  public String solution(String s) {
      String answer = "";
      ArrayList<Character> upper = new ArrayList<Character>();
      ArrayList<Character> lower = new ArrayList<Character>();

      for(int i=0; i<s.length(); i++){
          if(s.charAt(i) >= 'A' && s.charAt(i) <= 'Z')
              upper.add(s.charAt(i));
          if(s.charAt(i) >= 'a' && s.charAt(i) <= 'z')
              lower.add(s.charAt(i));
      }

      Collections.sort(upper);
      Collections.sort(lower);

      for(int i=lower.size()-1; i>-1; i--)
          answer += lower.get(i);

      for(int i=upper.size()-1; i>-1; i--)
          answer += upper.get(i);

      return answer;
  }
}

File: JavaScript/level1/문자열_내림차순으로_배치하기.js
function solution(s) {
    var answer = '';
    let array = [];
    
    for(let i=0; i<s.length; i++)
        array.push(s[i]);
    
    array.sort(function(a, b){
        return b.charCodeAt(0)-a.charCodeAt(0);
    })
    
    for(let i=0; i<array.length; i++)
        answer += array[i];
    
    return answer;
}